src/Components/Radio/Radio.component.jsx

```diff
@@ -77,14 +78,29 @@ const Radio = ({activeTrack, setActiveTrack}) => {
   const [status, setStatus] = useState(0);
   const [audio, setAudio] = useState(null);
   const [playing, setPlaying] = useState(false);
-  let nowPlaying = '<Song Name>';
-
-  if (activeTrack) {
-    nowPlaying = activeTrack.title;
-  }
+  //let nowPlaying = '<Song Name>';
+  const [nowPlaying, setNowPlaying] = useState('<CLICK PLAY!>');
+
+  const siteUrl = 'https://icecast.softboys.club:18000';
+  const fetchData = async () => {
```

Comment:
blocker (revision needed before merge)

this function needs to be declared in the useEffect hook, youll see as you include the appropriate dependency arrays

Reply:
`useEffect(()=>{}, [playing])` warns that fetchData be added to the dependencies. adding fetchData warns that 

>   Line 88:9:  The 'fetchData' function makes the dependencies of useEffect Hook (at line 144) change on every render. Move it inside the useEffect callback. Alternatively, wrap the 'fetchData' definition into its own useCallback() Hook  react-hooks/exhaustive-deps

Reply:
declaring the function in the useEffect means literally put the fetchData code inside the useEffect code.

Reply:
added fetchdata into the useEffect scope